yazi-config/preset/keymap-default.toml

```
@@ -232,12 +232,13 @@ keymap = [
 	{ on = "<C-[>",   run = "escape",         desc = "Go back the normal mode, or cancel input" },
 
 	# Mode
-	{ on = "i", run = "insert",                              desc = "Enter insert mode" },
-	{ on = "a", run = "insert --append",                     desc = "Enter append mode" },
-	{ on = "I", run = [ "move -999", "insert" ],             desc = "Move to the BOL, and enter insert mode" },
-	{ on = "A", run = [ "move 999", "insert --append" ],     desc = "Move to the EOL, and enter append mode" },
-	{ on = "v", run = "visual",                              desc = "Enter visual mode" },
-	{ on = "V", run = [ "move -999", "visual", "move 999" ], desc = "Enter visual mode and select all" },
+	{ on = "i", run = "insert",                                    desc = "Enter insert mode" },
+	{ on = "a", run = "insert --append",                           desc = "Enter append mode" },
+	{ on = "I", run = [ "move first-char", "insert" ],             desc = "Move to the BOL, and enter insert mode" },
+	{ on = "A", run = [ "move eol", "insert --append" ],           desc = "Move to the EOL, and enter append mode" },
+	{ on = "v", run = "visual",                                    desc = "Enter visual mode" },
+	{ on = "V", run = [ "move first-char", "visual", "move eol" ], desc = "Enter visual mode and select all" },
```

Here we probably need to use `move bol` since we want to select the whole line (including spaces) when we click `V`:

```suggestion
	{ on = "V", run = [ "move bol", "visual", "move eol" ], desc = "Enter visual mode and select all" },
```